mix.exs

```diff
@@ -64,10 +64,15 @@ defmodule JSONAPI.Mixfile do
         "Mitchell Henke",
         "Jake Robers",
         "Sean Callan",
-        "James Herdman"
+        "James Herdman",
+        "Mathew Polzin"
       ],
       licenses: ["MIT"],
-      links: %{github: "https://github.com/jeregrine/jsonapi", docs: "http://hexdocs.pm/jsonapi/"}
+      files: ["lib", "mix.exs", "README*", "LICENSE*"],
```

Comment:
Am I missing anything here? 🤔

Reply:
I'm pretty new to publishing to Hex, but do we _want_ to override the default given the description "Defaults to standard project directories, so you usually don't need to set this property."

Reply:
Good callout @mattpolzin! I went to see what the latest and great defaults are and found these: https://github.com/hexpm/hex/blob/d78a23f617dda568cc11642ada459cc3b2683787/lib/hex/package.ex#L5-L6

Looks like we're covered 👍